lib/oxidized/hook/githubrepo.rb

```diff
@@ -0,0 +1,16 @@
+class GithubRepo < Oxidized::Hook
+  def validate_cfg!
+    cfg.has_key?('remote_repo') or raise 'remote_repo is required'
```

Comment:
@charlescng I raise more specific KeyError exception now. There is no defined interface for error behaviour in Oxidized::Hook parent class, it calls validate_cfg! on children instances only when it is defined.